uiowa/uiowa
# Intranet followup.

Resolves #4640 

# Changes
- Updates simple_sitemap from 4.0.0 to 4.1.1 to fix a schema error.
- Adds an intranet config form so webmasters can set custom messages.
- Adds tests for sitenow_intranet and updates some general test config.
- Splits a webmaster permission for sitenow_intranet and re-sorts existing split config.
- Creates a hidden theme setting to control display of login link, sets to TRUE for new and existing sites.

# How to test

Sync a site, then enable the sitenow_intranet module and import config: `drush en sitenow_intranet` && `drush cim`. It is necessary to enable the module first rather than just enable the split and import config because there is some setup happening in sitenow_intranet.install.

- As an anonymous user of sitenow_intranet, I cannot access sitemap.xml, articles RSS feeds, search page, taxonomy terms, etc.
- As an anonymous user of sitenow_intranet, I can access robots.txt and it is set to deny all user agents for all paths.
- As an anonymous user of sitenow_intranet, I cannot see top links or footer block information on the access denied page.
- As an anonymous user of sitenow_intranet, I do not see a login link in the footer.
- As a webmaster using sitenow_intranet, I can configure custom messages for the access denied page and see the changes immediately.
- As an editor/publisher/webmaster using sitenow_intranet, I do not have access to SiteImprove.
- As an anonymous user of a non-sitenow_intranet site, I can see the login link in the footer.
